.travis.yml

```diff
@@ -13,11 +13,17 @@ install: glide install
 # Note: $TRAVIS_TAG only works correctly outside of the quote marks
 script: GOARCH=$BUILD_ARCH GOOS=$BUILD_OS go build -ldflags '-X main.version='$TRAVIS_TAG
 before_deploy:
- - chmod +x edge-node-manager
- - mkdir -p $TRAVIS_TAG
- - mv edge-node-manager $TRAVIS_TAG/edge-node-manager-$TRAVIS_TAG-$BUILD_OS-$BUILD_ARCH
+- mkdir -p $TRAVIS_TAG
+- mv edge-node-manager $TRAVIS_TAG/edge-node-manager-$TRAVIS_TAG-$BUILD_OS-$BUILD_ARCH
 deploy:
-  provider: s3
+- provider: releases
+  api_key:
+    secure: HN4cpnRXGB52lK+Hie0jnO/vKTR6VOkgogp1fHuWeyrSDBjXuczvDZV232BXrGlvbkqycQrm47KIv6AQpJtWa+IBYt+wxTAlnc3/Wwcq+0r3Gdgq9gATGLpotxrgZRrFBf1YF8bgiGQuTZsVrXzQv2ZQxWd7gBfb1TYnBkBOGF0mCEkHbCtYo5OxmbuiVxhub0FDq3qXXlGNygyLtZzCQ6ZgndkDf6Vj6jw7sHsNiLbJ0KCyVDSoWMQifQLEgOOGRoGVkzQXtlZ8j9JCc7qsMLTkGhJaCnBJAHi1T7ZjY6b2HunoHxKkYvt1XtsDBQCcVyDGt5fZKAK5CC4xnCMyPW+J9QKByU/Y5rps6ENYMknMnuxPUb+5oOelHqwhTEIyXnLxlJ3bfTGMWlz/eniO1anNof+H4U4ymwth47gAugQwNvz/aX3a10+tHL7RxZZsJ8S85Gg46U66Fr9VPw302n18UBjCC/Yf0LZ8wwH3mARLLEi1yBfuvayHfRCsd8+a8HQ9XEMzU0xuimJR4oKy21Jv3n8rTBRlgplHZ2Og5Ns3y/xE+p7VR01ABWlMlaFjlzuQ2q/WgUqXaV5jVMSfxZ+yDgOR/1CIHPVH0829U20xsSFl9c9sqV7i5kemIWqoR4i/9y/77aYIBkje18rYdqSYkbQBQAB4tMOmD96Cppc=
+  file: ''
```

Comment:
How does this work with s3? Doesn't this cut an empty release? Where do we link to the releases on s3?